submissions/jimohKafayat/HTML/productpage.html

```diff
@@ -10,7 +10,9 @@ <h2><em>KAHF KOLLECTIONS</e></h2>
 
 <nav>
 	<p>CATEGORIES</p>
-	<ol>y
+	<ol>
+		<a href="../html-forms/contactform.html">Contact Us</a>
```

Comment:
This navigation links you added here is for your next task. if you add it now you won't have anything to do for the next task and you won't be able to proceed. Kindly remove them for now